.gitignore

```diff
@@ -1 +1 @@
-node_modules
+node_modules
```

Comment:
The coverage folder also should be in gitignore folder

Reply:
done